content/docs/apps/troubleshooting.md

```diff
@@ -42,9 +42,9 @@ Cloud Foundry will attempt to detect the buildpack to use with your app by exami
 
 If you're seeing errors installing any Python dependencies, check whether you have a `vendor/` directory in your app's root. The Python buildpack won't use PyPI if you have a `vendor/` directory, so you'll need to rename that directory to something else.
 
-## [DEA] Droplet Execution Phase
+## Potential causes of issues during Droplet Execution Phase [DEA]
```

Comment:
Any docs that mention the droplet execution agent (dea) are outdated, since we haven't used that backend since the switch to govcloud (and with it diego). We don't have dea logs anymore--the current set of log types is at https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html#format. I'm thinking we can reference `[CELL]` logs instead of `[DEA]` logs here.